src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -8760,13 +8784,17 @@ def applymap(
             return self._apply_snowpark_python_function_to_columns(func, kwargs)
 
         # Check if the function is a known numpy function that can be translated to Snowflake function.
-        sf_func = NUMPY_FUNCTION_TO_SNOWFLAKE_FUNCTION.get(func)
+        sf_func = NUMPY_UNIVERSAL_FUNCTION_TO_SNOWFLAKE_FUNCTION.get(func)
         if sf_func is not None:
             # TODO SNOW-1739034: remove pragma no cover when apply tests are enabled in CI
             return self._apply_snowpark_python_function_to_columns(
                 sf_func, kwargs
             )  # pragma: no cover
 
+        if func in (np.sum, np.min, np.max):
```

Comment:
What about other aggregate functions like mean, std, var, etc?

Reply:
mean, std, var are mapped to builtin functions when used in df.apply. But when applied element-wise I have skipped it for now. Ideally users should not be using aggregate functions in element wise apply methods as it doesn't do much (most likely a user error).

Reply:
Agreed. I just thought it's the same case for sum/min/max (when applied element-wise). So expanding that list would just be for completeness/consistency. But it's not necessary.

Reply:
I see, I skipped them because output type will be float so we need to add casting np.mean(<int>) -> float.